wfe/wfe.go

```diff
@@ -344,6 +353,31 @@ const (
 	unknownKey = "No registration exists matching provided key"
 )
 
+func verifyJWSKey(body string) (*jose.JsonWebKey, *jose.JsonWebSignature, error) {
```

Comment:
It verifies the key is valid... but yeah `extractJWSKey` probably makes more sense.